### PR TITLE
lib: bsdlib: PDN socket option translation

### PR DESCRIPTION
**Pull request by metc**
Add translation for SO_PDN_CONTEXT_ID socket option.

Signed-off-by: Christopher Métrailler <christopher.metrailler@nordicsemi.no>

### CODE REVIEW

**Comment by metc** [comment]
Requires https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/195

**Comment by metc** [comment]
cc @gbakke

**Comment by gbakke** [comment]
Thanks, @metc, LGTM =)

**Comment by metc** [comment]
Please DNM until nrfxlib PR ref is SHA (https://github.com/NordicPlayground/nrfxlib/pull/80)

**Comment by lemrey** [comment]
Piled up a commit to offload `SO_PDN_STATE`.

**Comment by lemrey** [comment]
@rlubos should we squash the manifest commit?
Right now I have only rebased it after the commit that adds the offloading.

**Comment by rlubos** [comment]
You can update the Zephyr SHA now: 57bca6dd70f67b50a0659ed379c10f4a5b758696
Just note that Zephyr SHA is also updated in #1206, so these two PRs need to stay in sync.

@lemrey I think the manifest update is non-intrusive in this case, there's no need to squash it. It should not introduce any failures.

**Comment by rlubos** [comment]
The nrfxlib SHA can be updated: 0f140413f675b802f408a547bf66e74ae6ac4ed1

File: west.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ manifest:
     - name: zephyr
       repo-path: fw-nrfconnect-zephyr
       west-commands: scripts/west-commands.yml
-      revision: 4ebad02e3e270169220a9959f8c9ea6ba88f2e28
+      revision: pull/207/head
```
Comment (rlubos) [inline west.yml]: The Zephyr SHA in the manifest file on master is now ahead of this PR, please remove this change.

**Comment by rlubos** [comment]
Any other blockers on this one, or can we remove DNM safely (and possibly merge right away)?

**Comment by metc** [comment]
> LGTM, @metc can you please double check?

LGTM. Thank you @lemrey

**Pull request merged**